spanreceiver/spanreceiver.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanreceiver
+
+import (
+	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+)
+
+// SpanReceiver is an interface that receives spans.
+type SpanReceiver interface {
+	ReceiveSpans(...*tracepb.Span) (*Acknowledgement, error)
```

Comment:
We also need to receive the Node. Even though we send it once we will cache that value and always make it available.

Reply:
Yeah, you read my mind, I was literally pushing that up as your comment came in :)